examples/src/app/events.js

```diff
@@ -1,37 +1,19 @@
 /**
- * @typedef {object} Loading
+ * @typedef {CustomEvent} LoadingEvent
```

Comment:
`LoadingEvent` doesn't seem to be typed any longer, but we can make use of the template:

```ts
interface CustomEvent<T = any> extends Event {
    /** Returns any custom data event was created with. Typically used for synthetic events. */
    readonly detail: T;
    /** @deprecated */
    initCustomEvent(type: string, bubbles?: boolean, cancelable?: boolean, detail?: T): void;
}
```
For example:

```js
/**
 * @typedef {object} LoadingEventDetail
 * @property {boolean} showDeviceSelector - Show device selector.
 */
/**
 * @typedef {CustomEvent<LoadingEventDetail>} LoadingEvent
 */
/** @type {LoadingEvent} */
const test = {
    detail: {
        showDeviceSelector: true
    }
}
```

Reply:
Oh okay thanks I replace these!